arbos/arbosState/arbosstate.go

```diff
@@ -282,7 +286,7 @@ func (state *ArbosState) UpgradeArbosVersion(
 		case 10:
 			ensure(state.l1PricingState.SetL1FeesAvailable(stateDB.GetBalance(
 				l1pricing.L1PricerFundsPoolAddress,
-			)))
+			).ToBig()))
 
 		case 11:
 			// Update the PerBatchGasCost to a more accurate value compared to the old v6 default.
```

Comment:
Below in the ArbOS 30 case, we should still be requiring that it's in debug mode for now